### PR TITLE
Fixed remote motioneye camera config bug. (#2449)

### LINKED ISSUE #2449: Not possible to add more than 3 cameras
I have tried to add a fourth camera but this one won't show up. Is this a known bug?
It is working fine with the current release 0.42.1 but not on the current dev. version.

### PR DESCRIPTION
**Pull request by tunkio**
Originally remote motioneye camera config generator caused camera loop break which resulted empty config list. Fixes #2449

### CODE REVIEW

**Comment by tunkio** [comment]
Ok, seems to work. 

There was problem that we calculate signature before encoding which resulted invalid signature at the remote server. Need to do more investigation later should we do some adjustment there.(post data encoding, or move signature calculator...)

File: motioneye/remote.py [hunk 1]
```diff
@@ -271,7 +272,7 @@ async def set_config(local_config, ui_config) -> Union[str, None]:
         password,
         p,
         method='POST',
-        data=ui_config,
+        data=ui_config.encode('utf-8'),
```
Comment (MichaIng) [inline motioneye/remote.py:275]:
```suggestion
        data=ui_config.encode(),
```
UTF-8 is the default.

**Comment by MichaIng** [comment]
Any idea why (the first) changes were required? Did we break this with a recent PR? The missing encoding, as always, is due to Python 2 > Python 3.

**Review by MichaIng — APPROVED** [review]

**Comment by tunkio** [comment]
Dunno, didn't check the changelog why there was improper handling of the return values and stuff. Was wondering my self that the previous implementation always broke the loop if remote motioneye camera have been set. This leaded to an empty camera config list.

**Pull request merged**